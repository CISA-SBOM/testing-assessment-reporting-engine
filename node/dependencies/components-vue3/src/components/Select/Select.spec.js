import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from './Select.vue'

describe('Select', () => {
  it('should match its default snapshot', () => {
    const wrapper = shallowMount(Component, {
      props: {
        modelValue: 1,
        options: [
          { id: 1, value: 1, text: 'Option 1' },
          { id: 2, value: 2, text: 'Option 2' },
          { id: 3, value: 3, text: 'Option 3' }
        ]
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
