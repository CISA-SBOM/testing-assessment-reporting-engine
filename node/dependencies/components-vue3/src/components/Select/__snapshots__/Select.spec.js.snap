// Vitest Snapshot v1

exports[`Select > should match its default snapshot 1`] = `
<select
  class="form-control"
  data-id="sds-select"
>
  <option
    disabled=""
    value=""
  />
  
  <option
    value="1"
  >
    Option 1
  </option>
  <option
    value="2"
  >
    Option 2
  </option>
  <option
    value="3"
  >
    Option 3
  </option>
  
</select>
`;
